packages/app-bridge/src/types/AssetChooser.ts

```diff
@@ -0,0 +1,10 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AssetChooserObjectType } from './Terrific';
+
+export type AssetChooserFilter = {
```

Comment:
Please use the existing types, we also don't allow every filters, you can find the one we allow in `AssetChooserOptions`

Reply:
Hey, I can remove this file and update `AssetChooserOptions` (in [`types/Terrific.ts`](https://github.com/Frontify/brand-sdk/blob/e948fb0b49cc638d0aa6e61b23ab5a123ebce36d/packages/app-bridge/src/types/Terrific.ts#L44)) type to look like this (to enforce the allowed filters)
```
export type AssetChooserOptions = {
    // ... existing fields ...
    filters?: {
        key: 'id' | 'ext' | 'object_type' | 'external_url';
        values?: (number | string)[];
        inverted?: boolean;
        containsText?: string;
    }[];
} & (
    // ... selectedValueId or selectedValueIds
);
```

Would that work?

Reply:
Done!